src/datachain/lib/image.py

```diff
@@ -1,17 +1,44 @@
 from typing import Callable, Optional, Union
 
 import torch
-from PIL import Image
+from PIL import Image as PILImage
+
+from datachain.lib.file import File, FileError, Image, ImageFile
+
+
+def image_info(file: Union[File, ImageFile]) -> Image:
+    """
+    Returns image file information.
+
+    Args:
+        file (ImageFile): Image file object.
+
+    Returns:
+        Image: Image file information.
+    """
+    file = file.as_image_file()
+
+    try:
+        with file.open() as stream:
```

Comment:
just to double check - is there guarantee that it is a stream (that we don't download it)

e.g. does it mean that users in reality will have to disable cahce and prefetch for such operations?

Reply:
We are using [File.open](https://github.com/iterative/datachain/blob/main/src/datachain/lib/file.py#L279-L292) here, if cache is disabled, no caching will be used and file will be downloaded using [our regular mechanics](https://github.com/iterative/datachain/blob/main/src/datachain/client/fsspec.py#L377-L386).

Reply:
right, but is cache enabled by default? is it possible to disable it specifically for this operation? so, that simple query that is fetching metadata doesn't trigger the full download by default? Disabling explicitly won't be an obvious thing to do to end users ...